sklearn/linear_model/tests/test_logistic.py

```diff
@@ -947,7 +947,8 @@ def test_warm_start():
                                              solver=solver,
                                              random_state=42, max_iter=100,
                                              fit_intercept=fit_intercept)
-                    clf.fit(X, y)
+                    with ignore_warnings():
```

Comment:
Seeing as `ConvergenceWarning` subclasses `UserWarning`, it would be nice to see that changed.